README.md

```diff
@@ -1008,6 +1008,7 @@ _Related: [Groupware](#groupware)_
 - [Sshwifty](https://github.com/nirui/sshwifty) - Sshwifty is a SSH and Telnet connector made for the Web. `AGPL-3.0` `Go/Docker`
 - [Teleport](https://goteleport.com/) - Certificate authority and access plane for SSH, Kubernetes, web applications, and databases. ([Source Code](https://github.com/gravitational/teleport)) `Apache-2.0` `Go`
 - [tmate](https://tmate.io/) - Instant terminal sharing. ([Source Code](https://github.com/tmate-io/tmate)) `ISC` `C`
+- [WeTTY](https://github.com/butlerx/wetty) - Terminal in browser over http/https. ([Source Code](https://github.com/butlerx/wetty)) `MIT` `Typescript`
```

Comment:
Congratulations on your contribution

A couple changes to be made:
- Please remove the duplicate source code link
- Please change the language tag to `Nodejs`

Reply:
Whoopsie, sorry! I've done that now - should be okay!